app/routes/git_tokens.py

```
@@ -1,204 +1,343 @@
+
+
+@router.get(
+    "/",
+    response_model=Dict[str, Any],
+    status_code=status.HTTP_200_OK,
+    summary="Get all git labels",
+    description="Retrieve a list of all git labels with masked token values",
+)
+async def get_git_labels(
+    current_user_id: AuthenticatedUserDTO = CurrentUser,
+    pagination: PaginationParams = Depends(),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)